src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Publish.targets

```
@@ -40,6 +40,8 @@
   <UsingTask TaskName="ComputeManagedAssembliesToCompileToNative" AssemblyFile="$(IlcBuildTasksPath)" />
   <Target Name="_ComputeAssembliesToCompileToNative" DependsOnTargets="$(IlcDynamicBuildPropertyDependencies)">
 
+    <Warning Condition="Exists($(UserRuntimeConfig))" Text="The published project has a runtimeconfig.template.json, but PublishAot doesn't support this configuration format. If possible, move the configuration to the project file using RuntimeHostConfigurationOption." />
```

```suggestion
    <Warning Condition="Exists($(UserRuntimeConfig))" Text="The published project has a runtimeconfig.template.json that is not supported by PublishAot. Move the configuration to the project file using RuntimeHostConfigurationOption." />
```
Are there common situations where it is not possible to use RuntimeHostConfigurationOption as a replacement?